src/ontology/nbo-edit.owl

```diff
@@ -9938,6 +9943,17 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ISBN 0199640394") obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
+AnnotationAssertion(terms:contributor obo:NBO_0070001 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070001 "2023-04-22T14:17:43Z"^^xsd:dateTime)
```

Comment:
You should be careful with the way you use dates. In #168 you use dc:date with `2023-04-22`. It does not matter what you do to me personally, but it should be consistent throughout.

Reply:
When I create a new class, it automatically adds a date with a long chain of gobbledegook, but when I have to do it manually I don't know what more to add than, e.g. 2023-04-22. Generally I will not date additions I make to existing things, indeed I hope this won't happen again anyway now I have configured things properly to add a date and Orcid automatically.